distribution/docs/src/main/resources/content/_managing/_configuring/hardening-solr-index.adoc

```diff
@@ -1,20 +1,55 @@
-:title: Hardening Solr Index
+:title: Hardening Solr
 :type: subConfiguration
 :status: published
 :parent: Configuring Solr
-:summary: Hardening Solr Index.
+:summary: Hardening Solr.
 :order: 02
 
 == {title}
 
-The following sections provide hardening guidance for Solr; however, they are to serve only as reference and other an additional security requirements may be added.
+The following sections provide hardening guidance for Solr; however,
+they are to serve only as reference and other an additional security requirements may be added.
 
-=== Hardening Solr Server Configuration
+=== Changing Default Solr Server Password
+
+The Solr server configuration included with the ${branding} uses
+a default password that is insecure. To harden the Solr server,
+the default password must be changed to a secure password.
+
+. Complete the ${branding} installation process.
+. ${branding} and the Solr server should be running.
+. Choose a password that meets to your organization's requirements for secure passwords.
+. Update the Sold server user's password to the new password you have chosen. Note that
+changing the password will prevent ${branding} from communicating with Solr server
+until the ${branding} configuration is updated in
+subsequent steps.
+  .. Use a utility like curl to send an HTTP request to the Solr server
+  change the password.
+
+  curl --insecure --user ddf:changeit http://localhost:8983/solr/admin/authentication
+  -H 'Content-type:application/json'
+  -d '{ "set-user": {"ddf" : "PASSWORD"}}'
```

Comment:
In my windows faux terminal I had to modify the curl command to the following:
`curl -k -u "ddf:changeit" "https://localhost:8994/solr/admin/authentication" -H 'Content-type:application/json' -d "{ 'set-user': {'ddf' : 'pass'}}"`
Adding the quotes around the user credentials and reversing the quotes in the json.